app/services/MetricsService.scala

```diff
@@ -56,4 +60,8 @@ class MetricsServiceImpl @Inject()(metrics: Metrics) extends MetricsService {
 
   val postSetupPaymentsJourneyTimer: Timer = metrics.defaultRegistry.timer("post-setup-payments-journey-payment-api-timer")
   val postSetupPaymentsJourneyCounter: Counter = metrics.defaultRegistry.counter("post-setup-payments-journey-payment-api-failure-counter")
+
+  override val getDirectDebitStatusTimer: Timer = metrics.defaultRegistry.timer("get-direct-debit-status-from-financial-transactions-api-timer")
```

Comment:
Can you please remove override on this line and for the `val getDirectDebitStatusFailureCounter`